e2e/nav.spec.js

```diff
@@ -0,0 +1,68 @@
+require('coffeescript').register();
+_ = require('lodash');
+assert = require('should/as-function');
+
+testUtil = require('../../heap/test/util');
+rnTestUtil = require('./rnTestUtilities');
+
+const delay = async () => {
+  await new Promise(resolve => setTimeout(resolve, 500));
+};
+
+const doTestActions = async () => {
+  // Open the PropExtraction tab in the tab navigator.
+  await element(by.id('Nav')).tap();
+
+  await expect(element(by.id('navigate_stack'))).toBeVisible();
+  await element(by.id('navigate_stack')).tap();
+  await delay();
+  await expect(element(by.id('pop1'))).toBeVisible();
+  await element(by.id('pop1')).tap();
+  await delay();
+  await expect(element(by.id('navigate_modal'))).toBeVisible();
+  await element(by.id('navigate_modal')).tap();
+  await delay();
+  await expect(element(by.id('pop2'))).toBeVisible();
+  await element(by.id('pop2')).tap();
+  await delay();
+
+  await element(by.id('navSentinel')).tap();
+};
+
+describe('Navigation', () => {
+  before(done => {
+    db.orm.connection.sharedRedis().flushall(done);
+  });
+
+  describe('React Navigation autotrack', () => {
+    before(async () => {
+      await doTestActions();
+      await rnTestUtil.pollForSentinel('Nav');
+    });
+
+    it('tracks a tab navigator action', async () => {
+      await rnTestUtil.assertNavigationEvent(
+        'Nav::MainStack::Base',
+        'Navigation/NAVIGATE'
+      );
+    });
+    it('tracks a stack navigator action', async () => {
```

Comment:
Lil nit but probably want newline above this line for consistency.

Reply:
Oh yeah!  Thanks.  It's actually funny - I've got a jest runner plugin for VSCode that automatically inserts one-off "debug THIS test" buttons whenever it sees test-like stuff in code.  I've attached a screenshot of why I didn't think to put a newline here :-P
![Screen Shot 2019-03-14 at 4 40 30 PM](https://user-images.githubusercontent.com/1160174/54393407-12fb7500-4678-11e9-947f-100202c2957f.png)
So that "Debug" button _looks_ like a newline, but isn't!